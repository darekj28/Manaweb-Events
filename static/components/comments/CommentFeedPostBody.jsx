class CommentFeedPostBody extends React.Component {
	render() {
		var post = this.props.isOriginalPost ? (<h2>{this.props.content}</h2>) : <div>{this.props.content}</div>;
		return(
			<div className="CommentFeedPostBody">
				{post}
			</div>
		)
	}
}