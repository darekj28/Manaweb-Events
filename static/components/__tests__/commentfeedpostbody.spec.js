import CommentFeedPostBody from "../CommentFeedPostBody.jsx";

describe("CommentFeedPostBody", () => {
	it("renders correctly for original post", () => {
		const cfpb = renderer.create(
			<CommentFeedPostBody content="a" isOriginalPost={true}/>
			);
		expect(cfpb).toMatchSnapshot();
	});
	it("renders correctly for not original post", () => {
		const cfpb = renderer.create(
			<CommentFeedPostBody content="a" isOriginalPost={false}/>
			);
		expect(cfpb).toMatchSnapshot();
	});
});