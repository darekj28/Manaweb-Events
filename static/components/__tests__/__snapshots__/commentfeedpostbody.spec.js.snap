exports[`CommentFeedPostBody renders correctly for not original post 1`] = `
<div
  className="CommentFeedPostBody">
  <div>
    a
  </div>
</div>
`;

exports[`CommentFeedPostBody renders correctly for original post 1`] = `
<div
  className="CommentFeedPostBody">
  <h2>
    a
  </h2>
</div>
`;
